master/buildbot/changes/gerritchangesource.py

```diff
@@ -82,33 +84,37 @@ def checkConfig(self,
     def reconfigService(self,
                         gitBaseURL=None,
                         handled_events=("patchset-created", "ref-updated"),
+                        get_files=False,
                         debug=False):
         self.gitBaseURL = gitBaseURL
         self.handled_events = list(handled_events)
+        self.get_files = get_files
```

Comment:
while we have plenty of "let's have it public data members", i do not think it's a good idea to introduce new ones: please use `self._get_files`

Reply:
Roger that. I was trying to be consistent with the surrounding code (which is a bit of a challenge given that the buildbot sources feature every possible formatting and naming convention out there :)